pystiche_papers/johnson_alahi_li_2016/_data.py

```diff
@@ -68,7 +68,12 @@ def get_style_edge_size(
     def get_default_edge_size() -> int:
         if not impl_params:
             return 256
-
+        """
+        image_size:
+        https://github.com/pmeier/fast-neural-style/blob/813c83441953ead2adb3f65f4cc2d5599d735fa7/slow_neural_style.lua#L20
+        style:
+        https://github.com/pmeier/fast-neural-style/blob/813c83441953ead2adb3f65f4cc2d5599d735fa7/slow_neural_style.lua#L35
+        """
```

Comment:
1. remove the multiline string
2. Both links point to a size of 512.

Reply:
Point 2 is correct, because I did not find anything about the `image_size` 384. Should this actually be twice as large and justified by [source](https://github.com/pmeier/fast-neural-style/blob/813c83441953ead2adb3f65f4cc2d5599d735fa7/fast_neural_style.lua#L21)?

Reply:
I'll look into it.

Reply:
This will be resolved in #140. You will need to rebase after it is merged.